Makefile

```diff
@@ -18,12 +19,12 @@ dependencies:
 	luarocks make --local rockspec
 
 build:
-	$(DOCKER_COMPOSE) build
+	$(S2I) build . openresty-builder docker-gateway-test
```

Comment:
Ok, we can publish the openresty-builder manually.. then look for a way to automate it.. just to unblock this.